source/extensions/filters/http/response_map/config.cc

```diff
@@ -19,6 +19,14 @@ Http::FilterFactoryCb ResponseMapFilterFactory::createFilterFactoryFromProtoType
   };
 }
 
+Router::RouteSpecificFilterConfigConstSharedPtr
+ResponseMapFilterFactory::createRouteSpecificFilterConfigTyped(
+    const envoy::extensions::filters::http::response_map::v3::ResponseMapPerRoute&
+        proto_config,
+    Server::Configuration::ServerFactoryContext&, ProtobufMessage::ValidationVisitor&) {
+  return std::make_shared<FilterConfigPerRoute>(proto_config);
+}
+
```

Comment:
As simple as this block is, I found it hard to read.  I guess that's just to be expected when you have type names that are 71 characters log.  And I guess that's just to be expected with C++/Envoy :/ 

Does 1 arg per line make it better?

```C++
Router::RouteSpecificFilterConfigConstSharedPtr
ResponseMapFilterFactory::createRouteSpecificFilterConfigTyped(
    const envoy::extensions::filters::http::response_map::v3::ResponseMapPerRoute& proto_config,
    Server::Configuration::ServerFactoryContext&,
    ProtobufMessage::ValidationVisitor&) {
  return std::make_shared<FilterConfigPerRoute>(proto_config);
}
```

Maybe? But does the long `proto_config` line run afoul of some automated style checker?